assets/js/components/setup/CompatibilityChecks/CompatibilityErrorNotice.js

```diff
@@ -119,25 +124,36 @@ export default function CompatibilityErrorNotice( { error } ) {
 					) }
 					{ installed && (
 						<span>
-							{ __(
-								'Looks like this may be a staging environment and you already have the helper plugin. Before you can use Site Kit, please make sure you’ve provided the necessary credentials in the Authentication section and verified your production site in Search Console.',
-								'google-site-kit'
+							{ createInterpolateElement(
+								__(
+									'Looks like this may be a staging environment and you already have the helper plugin. Before you can use Site Kit, please make sure you’ve provided the necessary credentials in the Authentication section and verified your production site in Search Console. <GetHelpLink />',
+									'google-site-kit'
+								),
+								{
+									GetHelpLink: (
+										<Link
+											{ ...helperCTA(
+												developerPlugin,
+												documentationURL
+											) }
+										/>
+									),
+								}
 							) }
 						</span>
-					) }{ ' ' }
-					<Link
-						{ ...helperCTA( developerPlugin, documentationURL ) }
-					/>
+					) }
 				</p>
 			);
 		case ERROR_TOKEN_MISMATCH:
 			return (
 				<p>
-					{ __(
-						'Looks like Site Kit is unable to place or detect tags on your site. This can be caused by using certain caching or maintenance mode plugins or your site’s frontend is configured on a different host or infrastructure than your administration dashboard.',
-						'google-site-kit'
-					) }{ ' ' }
-					<GetHelpLink errorCode={ error } />
+					{ createInterpolateElement(
+						__(
+							'Looks like Site Kit is unable to place or detect tags on your site. This can be caused by using certain caching or maintenance mode plugins or your site’s frontend is configured on a different host or infrastructure than your administration dashboard.',
```

Comment:
```suggestion
							'Looks like Site Kit is unable to place or detect tags on your site. This can be caused by using certain caching or maintenance mode plugins or your site’s frontend is configured on a different host or infrastructure than your administration dashboard. <GetHelpLink />',
```

Reply:
Argh, thanks! 😆